src/Ship.java

```diff
@@ -27,10 +28,16 @@ public int getLife() {
 	//	a peice of logic is missing here
 	//
 	public void hit() throws Exception {
-		if (!this.sunk) {
+		
```

Comment:
logic is extremely inefficient and breaks the unit test stating this needs to throw an exception